cmd/get/apps/printer.go

```diff
@@ -71,6 +71,7 @@ func getTable(appResource app.Resource) *metav1.Table {
 		{Name: "Version", Type: "string"},
 		{Name: "Last Deployed", Type: "string", Format: "date-time"},
 		{Name: "Status", Type: "string"},
+		{Name: "Notes", Type: "string"},
```

Comment:
Can we name it _Reason_ instead? _Notes_ sounds a bit too generic.

Reply:
It felt weird for me as it is a reason for what? I understand the CRD contains that, but it is quite confusing in my opinion as the other fields reflect the actual status of Helm release and the reason *may* contain a fabricated message on the result of the last attempt on a Helm release.

Reply:
I agree that REASON would be strange in this context. NOTES works for me.